Added Functions to Retrieve Gyro Data

thesprintingturtle opened the pull request:
Added double array yawPitchRoll to DriveSub that is filled with the Pigeon IMU's yaw (0), pitch (1), and roll (2). Added function getYaw that updates yawPitchRoll and returns Yaw.

Commit:
Added Functions to Retrieve Gyro Data

Added double array yawPitchRoll to DriveSub that is filled with the Pigeon IMU's yaw (0), pitch (1), and roll (2). Added function getYaw that updates yawPitchRoll and returns Yaw.

coderkevin commented on 2019RobotCode/src/main/java/frc/robot/Susbsystems/DriveSub.java:
```diff
@@ -47,4 +52,11 @@ public void drive(double spd, double trn){
 
+  public double getYaw(){
+    gyro.getYawPitchRoll(yawPitchRoll);
+    if(yawPitchRoll[0] >= 180) yawPitchRoll[0] -= 360;
```
Let's not do single line if statements. Add braces.

coderkevin commented on 2019RobotCode/src/main/java/frc/robot/Susbsystems/DriveSub.java:
```diff
@@ -30,6 +32,9 @@
   public DifferentialDrive drive = new DifferentialDrive(right, left);
 
+  public PigeonIMU gyro = new PigeonIMU(6);
+  public double[] yawPitchRoll = new double[3];
```
These should be private too. No reason to have them public especially since you expose a public method below.

coderkevin reviewed (approved):
Just a couple notes. Pre-approving. Feel free to merge after fixing those things.

Commit:
Fixed Problems with Gyro Code

Changed gyro to private and yawPitchRoll to private. Changed single-line if statements in getYaw to use brackets.
